osu.Game/GameModes/Play/PlaySongSelect.cs

```diff
@@ -44,13 +45,22 @@ public PlaySongSelect(BeatmapDatabase database = null)
 
             const float scrollWidth = 640;
             const float bottomToolHeight = 50;
+            const float topToolHeight = 50;
             Children = new Drawable[]
             {
+                background = new Sprite {
+                    RelativeSizeAxes = Axes.Both,
+                    RelativePositionAxes = Axes.Both,
+                    Anchor = Anchor.Centre,
+                    Origin = Anchor.Centre,
+                    Size = Vector2.One,

```

Comment:
Not sure that we need to specify Size here. As far as I understand, RelativeSizeAxes should be enough.